messages/env.compute.collaborator.add.md

```diff
@@ -0,0 +1,13 @@
+# summary
+
+Add heroku user to all compute environments.
```

Comment:
The copy should probably be something along the lines of:
'Add a Heroku user as a collaborator on compute environments, allowing them to attach add-ons'

But we should really be getting some guidance on copy here. I'd ask Shyam for help with this. Or @jshackell-sfdc !

Reply:
I like Milad's copy.  Here's a teensy edit:

"Add a Heroku user as a collaborator on compute environments, allowing them to attach add-ons."